sources/it/82/statewide.json

```diff
@@ -8,24 +8,21 @@
         "country": "it",
         "state": "82"
     },
-    "data": "https://s3.amazonaws.com/data.openaddresses.io/cache/uploads/iandees/6dae2f/sicily20160804.csv.zip",
+    "data": "http://map.sitr.regione.sicilia.it/ArcGIS/rest/services/CART_2000/StradeEdifici_2000/MapServer/0/",
     "website": "http://www.sitr.regione.sicilia.it/geoportale/it/metadata/details/633",
     "license": {
         "url": "https://creativecommons.org/licenses/by-sa/3.0/",
         "attribution": true,
         "share-alike": true
         },
-    "type": "http",
-    "compression": "zip",
+    "type": "ESRI",
     "language": "it",
     "conform": {
         "number": ["CIVICO","ESPONENTE"],
```

Comment:
For Italy, the "esponente" is separated with a "/" usually instead of a space. This ought to do it:

```json
"number": {
    "function": "format",
    "fields": ["CIVICO", "ESPONENTE"],
    "format": "$1/$2"
},
```

With the format function, if "ESPONENTE" is blank, the "/" will not be used.

Otherwise looks good.